src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -1958,23 +1974,23 @@ static int do_list_mapped_devices(const std::string &format, bool pretty_format)
   Config cfg;
   NBDListIterator it;
   while (it.get(&cfg)) {
+    std::string snap = (cfg.snapid != CEPH_NOSNAP) ?
+      "@" + std::to_string(cfg.snapid) : cfg.snapname;
```

Comment:
This is a repeat of the previous suggestion, so I guess I need to explain ;)
- It doesn't make sense to parenthesize a ternary if condition because `?:` has one of the lowest precedences anyway (only `,` is lower in C++).  However it's a good habit to parenthesize the entire ternary if even where that appears to be redundant because it can help to avoid subtle bugs when the code is refactored.
- The "wrap" indentation is generally 4 spaces.
```suggestion
    std::string snap = (cfg.snapid != CEPH_NOSNAP ?
        "@" + std::to_string(cfg.snapid) : cfg.snapname);
```

Reply:
Makesense, thanks for highlighting it :-)